content/en/deployment_gates/setup.md

```diff
@@ -0,0 +1,372 @@
+---
+title: Set Up Deployment Gates
```

Comment:
Nit: this is referenced with lowercase U from the main page. I'd either put this in lowercase or make the links uppercase

```suggestion
title: Set up Deployment Gates
```